### ISSUE TITLE
Spotify not active. Exiting.

### ISSUE THREAD

**Issue by tialis** [SecUpwN/Spotify-AdKiller#39]
Hi all,

I followed the readme file (installed dependencies, modified ~/.config/spotify/User/<your username>/prefs and all) using Ubuntu 14.04, and the script is not working. 

On debug mode, it prompts:

## Using cvlc for local playback ##
Spotify not active. Exiting.
## Restoring settings ##

Is there any chance to fix the problem? 

Thanks!

**Comment by SecUpwN** [comment]
>Spotify not active. Exiting.

See section `Trobleshooting` at [Installation](https://github.com/SecUpwN/Spotify-AdKiller#installation). Which `Spotify` do you use? If you used [spotify-make](https://github.com/leamas/spotify-make), please symlink it (link `my-spotify` to `spotify`) so that our script recognizes the binary.

**Comment by tialis** [comment]
I think the problem is now solved. I executed in a terminal the spotify-adkiller.sh, and saw the following error:

wmctrl: command not found

I installed the wmctrl package and the .log shows that the macro is still running. Maybe the wmctrl package should be added to the dependencies for Ubuntu?

Now that the macro seems to be running, ads are still not being muted. I have the same awk errors as bug number #37. I'll stay tuned.

Thanks!
